packages/react/src/components/Popup/Popup.tsx

```diff
@@ -170,10 +175,12 @@ const findSiblingNodes = (
  * This adds accessibility to Dialog and Panel controls
  */
 export const Popup: React.FunctionComponent<IPopupProps> = React.forwardRef<HTMLDivElement, IPopupProps>(
-  (props, forwardedRef) => {
-    // Default props
-    // eslint-disable-next-line deprecation/deprecation
-    props = { shouldRestoreFocus: true, ...props };
+  (propsWithoutDefaults, forwardedRef) => {
+    const props = getPropsWithDefaults(
```

Comment:
FYI I just noticed (and pushed a fix for) an issue with how the defaults were provided here: since the values are actually used in other functions, just setting them here won't do anything. Instead it's necessary to use the `getPropsWithDefaults` helper to generate a props object that includes the defaults, and pass that through.

Reply:
This could really also use a test (Popup currently has no tests 😬) -- that would have caught the fact that the way the default was being set wouldn't do anything. But that can probably be a separate PR.